COLLADAMaya/src/COLLADAMayaDocumentExporter.cpp

```diff
@@ -335,6 +335,35 @@ namespace COLLADAMaya
             COLLADASW::URI sourceFileUri ( COLLADASW::URI::nativePathToUri ( currentScene ) );
             if ( sourceFileUri.getScheme ().empty () )
                 sourceFileUri.setScheme ( COLLADASW::URI::SCHEME_FILE );
+			if (sourceFileUri.scheme() == COLLADASW::URI::SCHEME_FILE && ExportOptions::sourceDataFormat() != ExportOptions::Absolute)
+			{
+				COLLADABU::URI relativeTo;
+				switch (ExportOptions::sourceDataFormat())
+				{
+				case ExportOptions::RelativeToExportedDAE:
+					relativeTo = COLLADABU::URI::nativePathToUri(getFilename());
+					break;
+				case ExportOptions::RelativeToMayaProjectWorkspace:
+				{
+					MString workspace;
+					if (MGlobal::executeCommand(MString("workspace -q -rd;"), workspace))
+						relativeTo = COLLADABU::URI::nativePathToUri(String(workspace.asChar()));
+					else
+						MGlobal::displayError("Cannot query workspace. <source_data> will be absolute.");
+				}
+					break;
```

Comment:
Is this default VS formatting ? Looks weird to me
Thought it was Github which was messed up due to tab size ;)
```
switch (value)
{
    case option1:
    {
        // do stuff
        break;
    }

    default:
    {
        // do default stuff
    }
}
```